myapp/spec/models/task_model_spec.rb

```diff
@@ -9,77 +9,49 @@
     let(:params) { { name: name, desc: desc, status: status, label: label, priority: priority } }
```

Comment:
【just some explanation】What I meant is to set the correct params here.
Then just update part of the parameter inside each "context" like in L17, L24...

But this current way also work!

Reply:
ohh OK, will do that in next PR then.